RELEASES.md

```diff
@@ -110,11 +110,49 @@ python -m pip install \
 
 ### Installing PyTorch Python packages
 
-Coming soon!
+> [!WARNING]
+> This is under **active** development.
+
+Using the index pages listed above, you can install `torch==2.7.0a0` instead of
+`rocm[libraries,devel]`.
+
+For example, with gfx110X-dgpu:
+
+```bash
+python -m pip install \
+  --pre --find-links https://therock-nightly-python.s3.amazonaws.com/gfx110X-dgpu/index.html \
+  torch==2.7.0a0
+
+pip freeze
+# ...
+# rocm-sdk==6.5.0rc20250606
+# rocm-sdk-core==6.5.0rc20250606
+# rocm-sdk-libraries-gfx110X-dgpu==6.5.0rc20250606
+# ...
+# torch==2.7.0a0+rocmsdk20250608
+```
 
-<!-- TODO: add `torch` to install commands
-       * needs new build to be compatible with 'rocm' instead of 'rocm-sdk'
-       * For 'rocm-sdk', need an environment workaround -->
+Current known issues (see also https://github.com/ROCm/TheRock/issues/808):
+
+- This currently fetches the old `rocm-sdk` package instead of the new `rocm`
+  package.
```

Comment:
Do you also want to point out that the ROCm version in the extra of the PyTorch wheel does not match the version if the ROCm packages fetched?

Reply:
Added to that issue:

> - [ ] Version metadata in `rocm` and `torch` wheels do not align

Reply:
Thanks for adding it to the issue.